Detectors/TPC/qc/include/TPCQC/PID.h

```diff
@@ -0,0 +1,62 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   PID.h
+/// @author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+///
+
+#ifndef AliceO2_TPC_ROC_H
+#define AliceO2_TPC_ROC_H
+
+//root includes
+#include "TH1F.h"
+//o2 includes
+#include "DataFormatsTPC/Defs.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "TPCBase/Sector.h"
+
+namespace o2
+{
+namespace tpc
+{
+namespace qc
+{
+
+/// Keep QC information for PID related observables
+///
+/// This is just a dummy implementation to get started with QC
+/// @author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+class PID
```

Comment:
A think a slightly more verbose name would be beneficial since PID is used in many contexts from reco, analysis, qc, and across detectors.
(Not sure if there is a standard convention across all QC tasks). Just some comment that isn't really blocking merging..

Reply:
Thanks for the comment. But then this would be a more general question I guess. I thought it is steered by namespace. e.g. all detectors implements Detectors.{h,cxx} in their namespace. Here it is o2::tpc::qc::PID. I thought this would be fine.

Reply:
Probably ok if we enforce the usage of namespaces.